dozuki.js

```diff
@@ -3,21 +3,24 @@
    function Dozuki(url, http) {
       baseUrl = url + "/api/2.0/";
       this.guides = {
-         get: function(guideid, langid, params) {
+         get: function(guideid, langid, params, headers) {
+            headers = headers || {};
             params = params || {};
             var url = baseUrl + "guides/" + guideid;
             if (langid) {
                params["langid"] = langid;
             }
 
+            if (!('X-ALLOW-HTTP' in headers)) {
+               headers['X-ALLOW-HTTP'] = 1;
```

Comment:
This seems odd..  if they don't provide the header, we automatically default it to 1? This allows you to bypass the https auto-redirect. I'd rather have it be secure by default, requiring this header to bypass the redirect.